spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/MqttPahoMessageDrivenChannelAdapter.java

```diff
@@ -199,7 +198,7 @@ private void connect() throws MqttException {
 			}
 			else {
 				IMqttAsyncClient theClient = clientManager.getClient();
-				Assert.state(theClient != null, "'client' must not be null");
+				Assert.state(theClient != null, "The 'client' must not be null, Consider to start 'clientManager'.");
```

Comment:
Let's fix the sentence!
```suggestion
				Assert.state(theClient != null, "The 'client' must not be null; consider to start the 'clientManager'.");
```